setup.py

```diff
@@ -0,0 +1,48 @@
+from io import open
```

Comment:
What do you think about using the built-in `open`?

Reply:
Changed!